src/test/java/uk/gov/hmcts/futurehearings/hmi/unit/testing/testsuites/RetrieveHearingsUnitTests.java

```diff
@@ -16,7 +24,10 @@
 
 import static io.restassured.RestAssured.given;
 import static uk.gov.hmcts.futurehearings.hmi.unit.testing.util.ResponseVerifier.*;
```

Comment:
ResponseVerifier as well please....

Reply:
We are using all the methods in the responseverifier class